### ISSUE TITLE
Hidden SVG.Text dmove causes recursion error in Firefox.

### ISSUE THREAD

**Issue by kinztechcom** [svgdotjs/svg.js#558]
When an SVG.Text is not currently rendered in the viewport on Firefox, and dmove is called on it, the page crashes and throws the following error:

```
Error: Script terminated by timeout at:
SVG.Element<.create@http://localhost:8080/js/libraries/svg.js:954:1
SVG.Text<.create@http://localhost:8080/js/libraries/svg.js:4313:5
SVG.adopt@http://localhost:8080/js/libraries/svg.js:132:15
assignNewId@http://localhost:8080/js/libraries/svg.js:5273:10
SVG.Element<.extend.clone@http://localhost:8080/js/libraries/svg.js:1016:19
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2077:23
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
SVG.BBox<.construct.bbox@http://localhost:8080/js/libraries/svg.js:2110:14
SVG.BBox<.create@http://localhost:8080/js/libraries/svg.js:2078:17
```

I fixed it locally by simply checking if the SVG.Text's display attribute was none before attempting to move it.

I'm using it in the following way:
Creation:

```js
this.text_svg = this.group.plain("").move(0, 0).font({ family: "Helvetica", size: 12 });
```

Moving:

```js
this.text_svg.dmove(x, y);
```

**Comment by Fuzzyma** [comment]
A few quetsions to this:

- does this also happen with text created with `this.group.text("").move(0, 0)`
- where is the display:none applied?

**Comment by kinztechcom** [comment]
Using `this.group.text("").move(0, 0)` still caused the recursion error.

I am rendering a rectangle with text in the middle for the area of the rectangle. If the area for the shape is turned off the text is hidden using the following line:
`this.text_svg.attr("display", "none");`

**Comment by Fuzzyma** [comment]
I am sorry but I cant reproduce this issue.

**Comment by Fuzzyma** [comment]
Albeit I never could reproduce this, it should be solved as part of #634 (part of the [3.0 branch](https://github.com/svgdotjs/svg.js/tree/3.0.0))
